integrity/typed_test.go

```diff
@@ -201,9 +201,12 @@ func TestTypedGet_Success(t *testing.T) {
 		nil,
 	).Return(response, nil)
 
+	var expectedModRevision int64 = 0
```

Comment:
It looks like the constant. Is it true?
```suggestion
	var expectedModRevision int64 = ModRevisionEmpty
```